src/components/landing/OfferingCards.js

```diff
@@ -0,0 +1,54 @@
+import OfferingCard from "./OfferingCard";
+import { FaUsers } from "react-icons/fa6";
+import { RiPresentationFill } from "react-icons/ri";
+import { GiPodium } from "react-icons/gi";
+import { LiaNewspaperSolid } from "react-icons/lia";
+
+const Offerings = [
+  {
+    title: "SEMINARS",
+    description:
+      "In-depth seminars offering insight into AI's latest technologies and advancements.",
+    icon: <FaUsers className="size-12 text-[#4F9AD7]" />,
+  },
+  {
+    title: "GUEST SPEAKERS",
+    description:
+      "Engaging sessions with AI experts, industry professionals, and prominent researchers.",
+    icon: <RiPresentationFill className="size-12 text-[#4F9AD7]" />,
+  },
+  {
+    title: "COMPETITIONS",
+    description:
+      "Challenge yourself and team up with peers in our AI competitions. A chance to put theory into practice.",
+    icon: <GiPodium className="size-12 text-[#4F9AD7]" />,
+  },
+  {
+    title: "AI NEWS AND OPPORTUNITIES",
+    description:
+      "Stay informed with the most recent developments, breakthroughs, and job opportunities in the AI realm.",
+    icon: <LiaNewspaperSolid className="size-12 text-[#4F9AD7]" />,
+  },
+];
+
+const OfferingCards = ({}) => {
+  return (
+    <section className="p-4">
+      <h2 className="text-4xl text-[#4F9AD7] text-center m-8">We Offer...</h2>
```

Comment:
do not use `<h2>` tags (it can mess with SEO)
instead use `<p>` tags with font sizes given in the tailwind docs